server/index.js

```diff
@@ -69,10 +72,10 @@ app.put('/api/description/:gameid', async (req,res) => {
 
 app.delete('/api/description/:gameid', async (req,res) => {
   const { gameid } = req.params;
-  if (gameid < 1 || gameid > 100) {
-    res.status(400).json({ error: 'Invalid game id' });
-    return;
-  }
+  // if (gameid < 1 || gameid > 100) {
```

Comment:
To be clear, though - no need to actually include these if they don't seem necessary.